README.md

```diff
@@ -0,0 +1,277 @@
+
+- npx stylelint "**/*.{css,scss}" for testing css errors
+- npx eslint . for Javascript errors
+
+
+### Deployment
+
+The app is deployed on Github pages. Follow the link below: https://stephenkati.github.io/First-capstone-project/
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHOR -->
+
+## 👥 Author <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Stephen Katuli**
+
+- GitHub: [@githubhandle](https://github.com/stephenkati)
+- Twitter: [@twitterhandle](https://twitter.com/Stephen57913145)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/stephen-katuli-a92752251/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[More backround images]**
+- [ ] **[Animations and transitions]**
+- [ ] **[More Javascript Functionality]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/stephenkati/First-capstone-project/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please reach out to me on my social media, hire me for more content like this.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+I would like to thankmy Microverse team and my coding partners for their support and guidance.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- project presentation -->
+
+
+##  PRESENTATION  <a name="presentation"></a>
+
+**PROJECT PRESENTATION VIDEO**
+- Follow this <a href="https://www.loom.com/share/6d4bce911db0406d9f5f246a3722c7f1">link</a> to see a brief presentation of the project.
+
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+- **[How good are your communication skills?]**
+
+  - [I have good writting and speaking communication skills in English.]
+
+- **[What else do you do apart from coding?]**
+
+  - [I do graphics design.]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is <a href="https://github.com/stephenkati/Global-Canvas-Art-Exhibition-Concert-/blob/features/License.md"> MIT  licensed.</a>
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
**[Optional]**
- [ ] This **NOTE** is just an instruction on how to make your repository licensed :thinking: Kindly remove it to make your readme file look clean and professional :blush:

Reply:
okay, on it right away